marketplace/commercelayer-sku/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "commercelayer-sku",
+  "version": "0.1.0",
```

Comment:
This should be `1.0` or things will get funky.

Reply:
Corrected.